src/mc/updateR.f03

```diff
@@ -19,6 +23,18 @@ subroutine updateR(I)
         call removeBead(wlc_bin,wlc_R_period(:,I),I)
     elseif (WLC_P__CONFINETYPE == 'sphere') then
         call removeBead(wlc_bin,wlc_R(:,I),I)
+    elseif (WLC_P__CONFINETYPE == 'cube') then
+        if (WLC_P__GJK_STERICS) then 
+            ! if bead i moves, then remove virtual beads i-1 and i
+            if (I > 1 .AND. I == left) then 
+                call removeBead(wlc_bin,wlc_R_GJK(:,I-1),I-1)
+            endif
+            if (I < WLC_P__NT) then 
+                call removeBead(wlc_bin,wlc_R_GJK(:,I),I)
+            endif
+        else
+            print*, "Not an option yet. See MCsim."
```

Comment:
This will through an error if I run a simultion with WLC_P__CONFINETYPE = "cube" and don't use sterics.  Many of my current simulations fit this criterion so this needs to be changed.

Reply:
great catch! sorry about that. fixing now